dotCMS/src/main/java/com/dotcms/rest/BundleResource.java

```diff
@@ -485,6 +529,19 @@ public Response deleteAllFail(@Context   final HttpServletRequest request,
                 "Removing bundles in a separated process, the result of the operation will be notified")).build();
     } // deleteAllFail.
 
+    private void sendDeleteResultsMessage(InitDataObject initData, Locale locale,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'locale' is not assigned and could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4652685)